modules/gen-grafana-creds.nix

```diff
@@ -0,0 +1,39 @@
+{ user ? null, password ? null }:
+
+let
+  pkgs = import (import ../fetch-nixpkgs.nix) {};
+in pkgs.stdenv.mkDerivation {
+  name = "gen-grafana-creds";
+  buildInputs = with pkgs; [ pwgen gnused ];
```

Comment:
Can drop gnused; this was carried over from gen-graylog-creds.nix, but isn't used here.